packages/webapp/pages/squads/index.tsx

```diff
@@ -1,46 +1,30 @@
-import React, { ReactElement, useContext } from 'react';
+import React, { ReactElement } from 'react';
```

Comment:
Let's revert the change here to avoid making any changes on the existing page. The deleted Card here should've been removed at the intermediate branch now.